lite2/client.go

```diff
@@ -547,7 +547,7 @@ func (c *Client) bisection(
 	case ErrNewValSetCantBeTrusted:
 		// continue bisection
 	default:
-		return errors.Wrapf(err, "failed to verify the header #%d ", newHeader.Height)
+		return errors.Wrapf(err, "failed to verify the header #%d", newHeader.Height)
 	}
 
 	if newHeader.Height == lastHeader.Height+1 {
```

Comment:
Is this check still needed? Since it cannot happen that we get `ErrNewValSetCantBeTrusted` with adjacent headers this could be handled by the `default:` above.

Reply:
Nope. Removed.